packages/vertical-layout/test/layout-improvements-polymer.test.js

```diff
@@ -0,0 +1,3 @@
+import './enable-layout-improvements.js';
+import '../vaadin-vertical-layout.js';
+import './layout-improvements.common.js';
```

Comment:
https://github.com/vaadin/web-components/pull/8561 was merged, so separate entrypoints for Lit and Polymer aren't necessary anymore.

Reply:
Thanks, changed to auto-generated Lit tests.